lib/test_dispatch/form.ex

```diff
@@ -13,19 +13,40 @@ defmodule TestDispatch.Form do
     fields = find_input_fields(form, "")
     selects = find_selects(form, "")
     textareas = find_textareas(form, "")
+    radio_buttons = find_radio_buttons(form, "")
 
-    Enum.uniq(fields ++ selects ++ textareas)
+    Enum.uniq(fields ++ selects ++ textareas ++ radio_buttons)
+  end
+
+  def find_radio_buttons(form, "") do
+    radios = Floki.find(form, "input[type=radio]")
+
+    checked = Enum.filter(radios, &floki_attribute(&1, "checked"))
+
+    checked_names = Floki.attribute(checked, "name")
+
+    other_radios =
+      form
+      |> Floki.find("input[type=radio]")
+      |> Enum.uniq_by(fn {_, list, _} ->
+        list |> Enum.find(fn {key, _} -> "name" == key end) |> elem(1)
+      end)
+      |> Enum.reject(fn radio -> floki_attribute(radio, "name") in checked_names end)
+
+    other_radios ++ checked
```

Comment:
Not sure, but maybe `split_with` works? A bit much is happening in this function
```suggestion
    [checked, rest] = 
      form
      |> Floki.find("input[type=radio]")
      |> Enum.split_with(&floki_attribute(&1, "checked"))

    checked_names = Floki.attribute(checked, "name")

    other_radios =
      rest
      |> Enum.uniq_by(fn {_, list, _} ->
        list |> Enum.find(fn {key, _} -> "name" == key end) |> elem(1)
      end)
      |> Enum.reject(fn radio -> floki_attribute(radio, "name") in checked_names end)

    other_radios ++ checked
```